PHP/proxy.php

```diff
@@ -716,15 +716,15 @@ public function canProcessRequest()
 
             //check with listed serverurl regardless if mustMatch is true or false
             foreach ($this->serverUrls as $key => $value) {
-                
                 $serverUrl = $value['serverurl'][0];
                 $serverUrl['url'] = $this->sanitizeUrl($serverUrl['url']); //Do all the URL cleanups and checks at once
-                
+				$serverUrl['matchall'] = strtolower((string) $serverUrl['matchall']);
+				
```

Comment:
I think the following 3 lines can be deleted with the `strtolower((string)...` change above.